### DISCUSSION TITLE
Response time / framerate issue

### DISCUSSION THREAD

**Discussion by dlynz** [prasanthrangan/hyprdots#981] [Q&A]
Hi does anyone have any ideas on any reason as to why in certain apps (discord, vscode, chrome) my frame rate doesn't feel up to 144 fps? I'm not sure if it's frame skipping or stuttering etc and it doesn't show up in recordings. When scrolling these apps don't seem to be very responsive to my inputs and it feels like I'm fighting to go anywhere.

GPU information for reference
![image](https://github.com/prasanthrangan/hyprdots/assets/100419289/fa76a9e6-9c15-42dc-b924-9e1b9d98ebe8)

Any ideas?

**Comment by Enigma1309** [comment]
they might be using xwalyand, try switching to their wayland alternatives, 
for discord --> webcord or armcord
i don't use either of the chrome or vscode, 
try switching to something like brave or firefox/librewolf, and use nvim instead of code idk if it will fix the issue but it works for me

**Reply by dlynz** [reply]
yea it doesnt really bother me anymore because i do use nvim and firefox more anyways it was just annoying but sometimes vscode is better for certain stuff so i end up using it

**Comment by kRHYME7** [comment]
try to run code with this commands

```
code --ozone-platform-hint=wayland --disable-gpu

code  --disable-gpu

code --ozone-platform-hint=wayland


```
This can be applied for some other chromium or electron base apps.  

also you can run ``` hyprctl clients ``` to know if an application is running on wayland or on X. 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/12a35db5-0828-4b14-b962-0b786a658ef2)

Maybe you need to adjust the flags inside the ./keybindings.conf for your key binds.
